src/smoothscroll.js

```diff
@@ -221,6 +222,12 @@ function polyfill() {
       method = scrollElement;
     }
 
+    const maxDistance = Math.max(Math.abs(x - startX), Math.abs(y - startY));
```

Comment:
Can we add a comment on top of this two lines of math operations (not that is unclear what they do), it will help people reading the code for the first time a lot, to understand the reasoning and why we are doing it.

Reply:
Sure, let put it in 2 lines and better for others to modify it in the future.

Reply:
Please, feel free to suggest a different explanation, comments, etc...
Also, I added the animation speed which is 1 pixel/ms. It will make it easier to modify it you think that another value matches better the native bahaivour.